larky/src/main/java/com/verygood/security/larky/modules/vgs/vault/DefaultVault.java

```diff
@@ -0,0 +1,184 @@
+package com.verygood.security.larky.modules.vgs.vault;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import com.verygood.security.larky.modules.vgs.vault.spi.LarkyVault;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.NoneType;
+import net.starlark.java.eval.Starlark;
+
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.regex.Pattern;
+
+
+public class DefaultVault implements LarkyVault {
+
+    private final Map<String, Object> persistentVaultStorage = new HashMap<>();
+    private final Map<String, Object> volatileVaultStorage = new HashMap<>();
+
+    private final Map<String, Map<String, Object>> storageConfig = new HashMap<String, Map<String, Object>>() {{
+        put("persistent", persistentVaultStorage);
+        put("volatile", volatileVaultStorage);
+    }};
+
+    private final Map<String, AliasGenerator> formatTokenizer = new HashMap<String, AliasGenerator>() {{
+        put("RAW_UUID", new RawAliasGenerator());
+        put("UUID", new UUIDAliasGenerator());
+        put("NUM_LENGTH_PRESERVING", new NumberLengthPreserving());
+
+        // Placeholders for unsupported formats found in
+        // https://www.verygoodsecurity.com/docs/terminology/nomenclature#alias-formats
+        put("PFPT", new NoopAliasGenerator("PFPT"));
```

Comment:
Can you attach a link to the Jira PBI to cover that functionality in the future, please?